app/models/care_plan.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from sqlalchemy import Column, ForeignKey, Integer, DateTime, String
+from sqlalchemy.orm import relationship
+
+from app.database import Base
+from .utils import ModelMixin
+
+
+class CarePlan(Base, ModelMixin):
+    __tablename__ = "careplans"
+
+    id = Column(Integer, primary_key=True, index=True)
+    date = Column(DateTime, default=datetime.datetime.today)
+    care_plan = Column(String(128), nullable=True)
```

Comment:
maybe we can from datetime import datetime and use it?

Reply:
we can) but I can import from this library also date and then so it will be inconvenient

Reply:
Thanks for your comments) They were relevant)